code/modules/surgery/organs/organ_external.dm

```diff
@@ -638,6 +633,21 @@ Note that amputating the affected organ does in fact remove the infection from t
 		owner.handle_splints()
 	return TRUE
 
+/obj/item/organ/external/proc/internal_bleed()
```

Comment:
Perhaps name it something like `cause_internal_bleeding()` or something? `internal_bleed` itself sounds more like a variable